izpack-panel/src/main/java/com/izforge/izpack/panels/path/PathInputPanel.java

```diff
@@ -216,11 +216,8 @@ else if (!checkCreateDirectory(file))
     public void panelActivate()
     {
         super.panelActivate();
-        if (modifyInstallation())
-        {
-            // installation directory has to exist if an installation is being modified
-            mustExist = true;
-        }
+        // installation directory has to exist if an installation is being modified
+        mustExist = modifyInstallation();
```

Comment:
There is a change in the existing logic. If the ```mustExist``` flag is set to ```true``` before ```panelActivate``` is called then the flag can be reset to ```false``` if  ```modifyInstallation()``` returns ```false```.

Reply:
Agreed, in this case we want to go with what `modifyInstallation()` returns. Hence, it is ok.